dev/com.ibm.ws.javaee.ddmodel/test/com/ibm/ws/javaee/ddmodel/DDJakarta11Elements.java

```diff
@@ -0,0 +1,424 @@
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.javaee.ddmodel;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+import org.junit.Assert;
+
+import com.ibm.ws.javaee.dd.common.ContextService;
+import com.ibm.ws.javaee.dd.common.Description;
+import com.ibm.ws.javaee.dd.common.JNDIEnvironmentRefs;
+import com.ibm.ws.javaee.dd.common.ManagedExecutor;
+import com.ibm.ws.javaee.dd.common.ManagedScheduledExecutor;
+import com.ibm.ws.javaee.dd.common.ManagedThreadFactory;
+import com.ibm.ws.javaee.dd.common.Property;
+
+/*
+ * jakartaee_11.xsd - jakartaee_10.xsd
+ *   <xsd:import namespace="http://www.w3.org/XML/1998/namespace"
+              schemaLocation="https://www.w3.org/2001/xml.xsd"/>
+
+<xsd:complexType name="context-serviceType">
+      <xsd:element name="qualifier"
+                   type="jakartaee:fully-qualified-classType"
+                   minOccurs="0"
+                   maxOccurs="unbounded">
+      </xsd:element>
+</xsd:complexType>
+
+<xsd:complexType name="managed-executorType">
+      <xsd:element name="virtual"
+                   type="jakartaee:true-falseType"
+                   minOccurs="0"
+                   maxOccurs="1">
+      </xsd:element>
+      <xsd:element name="qualifier"
+                   type="jakartaee:fully-qualified-classType"
+                   minOccurs="0"
+                   maxOccurs="unbounded">
+      </xsd:element>
+</xsd:complexType>
+
+<xsd:complexType name="managed-scheduled-executorType">
+      <xsd:element name="virtual"
+                   type="jakartaee:true-falseType"
+                   minOccurs="0"
+                   maxOccurs="1">
+      </xsd:element>
+      <xsd:element name="qualifier"
+                   type="jakartaee:fully-qualified-classType"
+                   minOccurs="0"
+                   maxOccurs="unbounded">
+      </xsd:element>
+</xsd:complexType>
+
+<xsd:complexType name="managed-thread-factoryType">
+      <xsd:element name="virtual"
+                   type="jakartaee:true-falseType"
+                   minOccurs="0"
+                   maxOccurs="1">
+      </xsd:element>
+      <xsd:element name="qualifier"
+                   type="jakartaee:fully-qualified-classType"
+                   minOccurs="0"
+                   maxOccurs="unbounded">
+      </xsd:element>
+</xsd:complexType>
+*/
+public class DDJakarta11Elements {
+
+    public static List<String> names(String... name0) {
+        List<String> names = new ArrayList<String>(6);
+        for (String name : names) {
+            names.add(name);
+        }
+        return names;
+    }
```

Comment:
The parameter to this method isn't used and the loop is a no-op.  Should it be this instead?
```suggestion
    public static List<String> names(String... name0) {
        List<String> names = new ArrayList<String>(name0 == null ? 0 : name0.length);
        if (name0 != null)
            for (String name : name0) {
                names.add(name);
            }
        return names;
    }
```

Reply:
This was copied over from the EE 10 element object. Fixing in next commit.